### ISSUE TITLE
Make experimentr users aware of privacy issues w/ Mechanical Turk ids

### ISSUE THREAD

**Issue by codementum** [codementum/experimentr#66]
- Bring in the paper that talks about identifying MTurk users via their IDs
- Provide guidelines for releasing experiment data (using the experiment id we already automatically generate)

**Comment by connorgr** [comment]
Might be worth defaulting Experimentr to use an anonymized hash (e.g.,
SHA1) for data analysis and export. Then include a warning whenever the
Turker IDs are used instead of hash.

**Comment by codementum** [comment]
Right-- Turk IDs are mainly (only?) used to prevent multiple attempts.

Exporting still needs some work, but when we revise it some sort of privacy warning / restriction would be a good feature.
